dgl_ptm/dgl_ptm/link_deletion.py

```diff
@@ -1,7 +1,70 @@
-#!/usr/bin/env python
-# coding: utf-8
+import dgl
+import torch
 
-# link_deletion - Randomly deletes links between agents
+def link_deletion(agent_graph, del_prob: float):
+    '''
+        link_deletion - Randomly deletes links between agents upto a deletion probablity 'del_prob'
```

Comment:
deletes links between agents with a deletion probability 'del_prob' by samoliing against a random uniform distribution.

Reply:
Changed in latest commit